qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -613,6 +615,24 @@ def __getitem__(self, key):
 
         return MatrixIterator(self)
 
+    def grouping(self):
```

Comment:
I'll split this to another PR, but for now this is necessary to simplify the code (to drop the dependency from opflow.AbelianGrouper).